workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,434 @@
+---
+name: 'Spin The Wheel'
+description: 'Build a random picker spinning wheel using HTML, CSS, and JavaScript'
+author: '@HariOm987'
+img: https://cloud-gvkfrv1qw.vercel.app/0image.png
+---
+
+In this workshop, you're going to make a colorful spinning wheel with your own custom content, in 20 minutes!
+
+Here are [live demo](https://movie-selector.hariom04.repl.co/) and the [final code](https://repl.it/@hariom04/movie-selector#index.html).
+
+![Final Output of our Workshop](https://cloud-4945tg5q1.vercel.app/0final.png)
+
+^^^ If you will follow this workshop strictly then you will make something like the above picture, but feel free to add up your imaginations with patterns and colors and you will end up with something better than this. So let's begin. 😁✨
+
+# Part 1: Prerequisites
+* Account on Repl.it
+* Basic knowledge of HTML, CSS and JavaScript would be helpful but is not necessary to continue further.
+
+# Part 2: Setup
+## Setting up our code environment
+We can use any code editor but keeping accessibility in our mind, we will be using Repl.it. It is an online code editor that is easy to use and can directly put your website in the real world.
```

Comment:
kind of unnecessarily long, i think you should just say "for this workshop, we'll be using repl.it, a free, online code editor."